Foundation/NSOrderedSet.swift

```diff
@@ -351,8 +351,12 @@ public class NSMutableOrderedSet : NSOrderedSet {
         }
     }
 
-    public init(capacity numItems: Int) {
-        super.init(objects: nil, count: 0)
+    public convenience init(capacity numItems: Int) {
+        self.init(array: [])
+    }
+
+    public override init(objects: UnsafePointer<AnyObject?>, count cnt: Int) {
```

Comment:
This designated initializer was missing, which meant the other convenience initializers were silently missing as well. That probably points to some potential area of improvement in the compiler.